atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -110,6 +111,7 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
     """
 
     EXCLUDE_FILENAME = 'exclude-platform'
+    UNREACHABLE_CLUSTER_LOAD = -1
```

Comment:
Nitpick: if this is not meant to be used as a number, consider using a symbol-like value:
```python
UNREACHABLE_CLUSTER_LOAD = object()
```